src/commands/appdistribution-distribute.ts

```diff
@@ -98,11 +103,39 @@ module.exports = new Command("appdistribution:distribute <distribution-file>")
       );
     }
 
+    if (
+      distribution.distributionFileType() === DistributionFileType.AAB &&
+      app.aabState !== AabState.ACTIVE &&
+      app.aabState !== AabState.AAB_STATE_UNAVAILABLE
+    ) {
+      switch (app.aabState) {
+        case AabState.PLAY_ACCOUNT_NOT_LINKED: {
+          throw new FirebaseError("This project is not linked to a GooglePlay account.", {
+            exit: 1,
```

Comment:
`exit: 1` is the default and does not need to be specified

Reply:
👍  - I may do a follow up change to clean more of these up. We seem to be about 90-10 exit: 1 vs not specifying